Modules/Sources/BuildSettingsKit/BuildSettings+Live.swift

```diff
@@ -39,6 +39,11 @@ private extension Bundle {
             fatalError("unexpected value: \(object) for key: \(key)")
         }
     }
+
+    func urlValue(forKey key: String) -> URL {
+        let urlWithoutScheme: String = infoValue(forKey: key)
+        return URL(string: "https://\(urlWithoutScheme)")!
+    }
```

Comment:
This could be improved with a fallback like if the strings already has "https://" at the start, don't prefix it with it.